ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
 
-set -o nounset
 set -o errexit
 set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
 expiration_date=$(date -d '4 hours' --iso=minutes --utc)
 
+if [[ -z "${INSTANCE_TYPE}" ]]; then
+  instance_type="m4.large"
+else
+  instance_type="${INSTANCE_TYPE}"
+fi
```

Comment:
From the discussion in the issue it doesn't sound to me like this is the approach we want to take: https://issues.redhat.com/browse/DPTP-1740
/hold